bool publish_map_odom_tf
bool publish_odom_base_link_tf
bool publish_map_odom_base_link_without_prefix
bool publish_world_tf
bool publish_world_tf_without_prefix
---
bool map_odom_tf_is_published
bool odom_base_link_tf_is_published
